asciidoc-confluence-publisher-converter/src/test/resources/inter-document-cross-references/include-page-with-title-in-include.adoc

```diff
@@ -0,0 +1 @@
+= {title-key}
```

Comment:
Is it relevant for the bug to occur that the title is set via a property? Or would the same issue arise even if the title is set here directly? If it is the latter, I would prefer to remove the property since it adds more complexity to the test that is not required to reproduce the issue.

Reply:
It is indeed not need. I'll remove it!